FWCore/Utilities/test/friendlyname_t.cppunit.cpp

```diff
@@ -37,6 +37,10 @@ void testfriendlyName::test() {
   classToFriendly.insert(Values("bar::Foo", "barFoo"));
   classToFriendly.insert(Values("std::vector<Foo>", "Foos"));
   classToFriendly.insert(Values("std::vector<bar::Foo>", "barFoos"));
+  classToFriendly.insert(Values("std::set<bar::Foo>", "barFoostdset"));
+  classToFriendly.insert(Values("std::map<Foo, bar::Bar>", "FoobarBarstdmap"));
```

Comment:
I added these two just to test the current behavior with `std::set` and `std::map`.